client/v1/src/main/java/org/opennms/velocloud/client/v1/VelocloudApiClientV1.java

```diff
@@ -266,4 +271,32 @@ public List<User> getUsers(final Integer enterpriseId) throws VelocloudApiExcept
             throw new VelocloudApiException("ApiException:" + e.getMessage());
         }
     }
+
+    @Override
+    public List<User> getEnterpriseProxyConnections() throws VelocloudApiException {
+        try {
+            final var partnerConnections =
+                    userMaintenanceApi.enterpriseProxyGetEnterpriseProxyUsers(new EnterpriseProxyGetEnterpriseProxyUsers());
```

Comment:
This is the same call as in `getUsers` but with a different filter?

Reply:
I don't think so. At least it doesn't appear that way from the end points.
From the API documentation for example there are this 2 endpoints
/enterprise/getEnterpriseUsers Gets all enterprise users for the specified enterprise
 /enterpriseProxy/getEnterpriseProxyUsers Gets all proxy admin users for the specified enterprise